60 age sex model to dev

alaparicioc opened the pull request.

doug-leasure committed:
rename simulation directory

alaparicioc committed:
Create age-sex-model.stan

alaparicioc committed:
Update age-sex-model.stan

alaparicioc committed:
Update age-sex-model.stan

alaparicioc committed:
Create 1_stan_model_data_config.R

alaparicioc committed:
Create README.txt

doug-leasure committed:
Merge pull request #72 from OxfordDemSci/dev

dev into age-sex branch

alaparicioc committed:
Create 3_comparison_DL_2023

alaparicioc committed:
Create 2_age_sex_model.stan

alaparicioc committed:
Create 1_mcmc_age-sex.R

adpated version fo 1_mcmc.R to include age and sex categories

alaparicioc committed:
Create 1_base_age-sex_model_config.R

alaparicioc committed:
Update 2_age_sex_model.stan

alaparicioc committed:
Create 2_age_sex_model.stan

This the the age-sex model with observed 10-year age groups (20-29, 30-39, 40-49, 50-59, 60+).

alaparicioc committed:
Create 2_age_sex_model_config.R

This is the file that configures data for the 2_age_sex_model.stan file

alaparicioc committed:
Update 1_mcmc_age-sex.R

updated paths to alt folder

alaparicioc committed:
Create 2_age_sex_model.stan

alaparicioc committed:
Create 2_age_sex_model_config.R

alaparicioc committed:
Update 2_age_sex_model_config.R

alaparicioc committed:
Update 2_age_sex_model_config.R

alaparicioc committed:
Update 2_age_sex_model_config.R

inclusion of new initial values, which makes chains running.

alaparicioc committed:
Update 2_age_sex_model_config.R

alaparicioc committed:
Create 3_age_sex_model.stan

This model is based on src/simulation/2_modelling/models/8_prop_model_RW expanding dimension of I to C=I*S*G (oblasts*sexes*age_groups)

alaparicioc committed:
Create 3_age_sex_model_config.R

This is the config file for running the modified version of src/simulation/2_modelling/models/8_prop_model_RW.stan that expands I to C=I*S*G

alaparicioc committed:
Update 3_age_sex_model.stan

alaparicioc committed:
Update 2_age_sex_model.stan

updated model with growth rate, adapting @njirons way of estimating p_G and modelling p_FG

alaparicioc committed:
Update 2_age_sex_model_config.R

alaparicioc committed:
Create 4_age_sex_model.stan

Stan model without FG _ratio and one covariate. There are no droppings (i.e. there are 27 oblast, 2 sexes, 4 age groups).

alaparicioc committed:
Update 4_age_sex_model.stan

alaparicioc committed:
Update 4_age_sex_model.stan

alaparicioc committed:
Create 4_age_sex_model_config.R

Preparation of data for 4_age_sex_model.stan

alaparicioc committed:
Organisation of files

alaparicioc committed:
Reorganising files from src/2_model/alt/models directory

alaparicioc committed:
Renaming files src/2_model/alt/4_age_sex_model_config.R

alaparicioc committed:
Renaming files  src/2_model/alt/4_age_sex_model.stan

alaparicioc committed:
Renaming files src/2_model/alt/3_age_sex_model_config.R

alaparicioc committed:
Renaming files src/2_model/alt/3_age_sex_model.stan

alaparicioc committed:
Renaming files src/2_model/alt/2_age_sex_model_config.R

alaparicioc committed:
Renaming files src/2_model/alt/2_age_sex_model.stan

alaparicioc committed:
Renaming files src/2_model/alt/23_age_sex_model_config.R

alaparicioc committed:
Renaming files src/2_model/alt/23_age_sex_model.stan

alaparicioc committed:
Renaming files src/2_model/alt/22_age_sex_model_config.R

alaparicioc committed:
Renaming files src/2_model/alt/22_age_sex_model.stan

alaparicioc committed:
Renaming files src/2_model/alt/21_age_sex_model_config.R

alaparicioc committed:
Renaming files src/2_model/alt/21_age_sex_model.stan

alaparicioc committed:
Create 211_age_sex_model.stan

alaparicioc committed:
Create 211_age_sex_model_config.R

alaparicioc committed:
Relocating src/2_model/alt/3_comparison_DL_2023

alaparicioc committed:
Update 2.1.cleaning_model_output.R

alaparicioc committed:
Update 2.1.cleaning_model_output.R

alaparicioc committed:
Update 2.1.cleaning_model_output.R

alaparicioc committed:
Update 2.1.cleaning_model_output.R

alaparicioc committed:
Delete src/2_model/models/2_age_sex_model.stan

alaparicioc committed:
Delete src/2_model/models/2_age_sex_model_config.R

alaparicioc committed:
Delete src/2_model/models/age-sex-model directory

alaparicioc committed:
Update 10_create_master_index.R

alaparicioc committed:
Update 10_create_master_index.R

alaparicioc committed:
Merge branch 'dev' into 60-age-sex-model

alaparicioc committed:
Merge pull request #106 from OxfordDemSci/dev

merging dev into age-sex model branch

edarin commented:
This pull request has conflict with the dev branch (see github comment).
Furthermore it is not clear what it is addressing. It would be great to have some form of modification summary, as well as, in the future, having better description of each commit (rather than "Update script", explaining what has been changed and what for).
A PR should be seen as a set of changes that are solving a problem.

alaparicioc committed:
Merge branch 'dev' into 60-age-sex-model

alaparicioc commented:
> This pull request has conflict with the dev branch (see github comment). Furthermore it is not clear what it is addressing. It would be great to have some form of modification summary, as well as, in the future, having better description of each commit (rather than "Update script", explaining what has been changed and what for). A PR should be seen as a set of changes that are solving a problem.

The PR is to merge age-sex model branch. Files should only modify the alt folder. I resolved the conflict that was related to the master index. I wanted to close this branch to split it up in issues with smaller tasks

edarin commented:
97 files have been changed (mainly in alt indeed): https://github.com/OxfordDemSci/UkraineNowPop/pull/108/files.
An entire folder `0_simulation` which does not exist in `dev` anymore is trying to be merged in.

alaparicioc committed:
Create 313_covs_model.stan

Model based on https://github.com/OxfordDemSci/UkraineNowPop/blob/d1d8f4333a4fdd0a955d90ded0bf8fb30f4a7c1a/src/2_model/models/3_covs_model.stan to estimate age-sex population stocks based on Facebook and Instagram.

doug-leasure reviewed (commented):
Thanks for opening this PR, @alaparicioc. 

I think it is a good idea to create some sub-issues for the age-sex model issue #60 to try to make it more manageable. 

However, I am hesitant to approve this pull request because it modifies so many files in addition to the files that you mentioned from `./src/2_model/alt/models_andrea/`. 

Some of the other changes look reasonable to me, like renaming `./src/simulation/` to `./src/0_simulation/`. But, there are many changed files that are the working models that originated from other members of the group, and it is difficult to review exactly what has changed in all of these files in the agesex branch. I am worried that by merging this pull request, we may unintentionally break some of those working models. 

Can you please create the sub-issue(s) from #60 that you have in mind and then create a branch (from dev) to work on one or more of those sub-issues? Then, you can copy your code from this branch into that new branch. This should allow us to keep the changes that you want to keep while avoiding unintentional changes to the code in dev.

alaparicioc committed:
Update 313_covs_model.stan

revised code

alaparicioc committed:
Create 322_covs_model

Updating model 322_covs_model based on 3_covs_model, only sex, with constraint, with covariates

alaparicioc committed:
relocating 313_covs_model

alaparicioc committed:
Rename 322_covs_model to 322_covs_model.stan

alaparicioc committed:
Rename src/2_model/alt/322_covs_model.stan to src/2_model/alt/322_covs_model/322_covs_model.stan
